src/main/java/com/google/api/codegen/CodeGeneratorTool.java

```diff
@@ -101,14 +101,14 @@ public static void main(String[] args) throws Exception {
 
   private static int generate(
       String descriptorSet,
-      String[] apiConfigs,
+      String[] configs,
```

Comment:
Right, but the tool option name itself is "configs". It's easier to just match them I think.